apps/google-analytics-4/lambda/src/controllers/apiController.ts

```diff
@@ -31,7 +31,10 @@ const ApiController = {
     try {
       const serviceAccountKeyFile = req.serviceAccountKey;
       const { propertyId, slug, startDate, endDate, dimensions, metrics } =
-        req.params as unknown as RunReportParamsType;
+        req.query as unknown as RunReportParamsType;
+
+      const formatArrays = (param: string | string[]) =>
+        Array.isArray(param) ? param : param.split(',');
```

Comment:
also wondering if there is an easier way to format arrays sent as query params. I dug thru the internet but am sure there is something a bit more succinct than this.